### DISCUSSION TITLE
Capabilities endpoint responses have changed

### DISCUSSION THREAD

**Discussion by MattiJarvinen** [tillsteinbach/VWsFriend#60] [General]
From what I gather in last few weeks capabilities endpoint responses have changed `https://vdbs.apps.emea.vwapps.io/v1/vehicles/<vin>/users/<guid>/capabilities` . Now they list API urls and possible scopes too.

Also VWsFriend started to give "ERROR:VWsFriend:Retrieval error during update. Will try again after configured interval of 300s" in the few hours.

![image](https://user-images.githubusercontent.com/12794435/134958545-d7a47cd4-a3de-4e7c-a8ae-6da72e711831.png)

**Comment by tillsteinbach** [comment]
Yes, the servers are responding with 500 internal server error right now, therefore the retrieval error.

I don't know the https://vdbs.apps.emea.vwapps.io domain yet. Where is that from? Is that from the my volkswagen webpage?

**Reply by MattiJarvinen** [reply]
It's from my volkswagen. Apparently from privacy settings portion. It now seems to load data for it when entering the page and at least listing new endpoints.

**Reply by tillsteinbach** [reply]
Ok, I may have a look tomorrow if it is useful. It may need different authentification though. I use the endpoints for the smartphone app instead. Do you see any endpoint with data that we don't know yet?

**Comment by MattiJarvinen** [comment]
Doesn't really seem like it at least there's no changes in the website UI.

ma 27. syysk. 2021 klo 22.22 Till Steinbach ***@***.***)
kirjoitti:

**Reply by tillsteinbach** [reply]
I had a look. There are some interesting endpoints, but unfortunately almost all are not working
